CI.md

```diff
@@ -8,6 +8,6 @@ GitHub Actions are used to run the Continuous Integration for amazon.aws collect
 | ------ |-------| ------ | -----------|
 | changelog |Checks for the presence of Changelog fragments | 3.9 | devel |
 | Linters | Runs `black` and `flake8` on plugins and tests | 3.9 | devel |
-| Sanity | Runs ansible sanity checks | 3.8, 3.9, 3.10, 3.11 | Stable-2.12, 2.13, 2.14 (not on py 3.11), Stable-2.15+ (not on 3.8) |
+| Sanity | Runs ansible sanity checks | 3.9, 3.10, 3.11 | Stable-2.12, 2.13, 2.14 (not on py 3.11), Stable-2.15+ (not on 3.8) |
```

Comment:
I may be wrong, but this information is completely outdated looking at the tests we're running in the CI right now. Can you please double-check and update at needed?

Reply:
updated.